dependencies.props

```diff
@@ -1,14 +1,71 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+
+  <!-- For detailed explanation of the reference versions see:
+
+        https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+        https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+       These ref versions are pulled from https://github.com/dotnet/versions.
+  -->
   <PropertyGroup>
-       <CoreCLRVersion>2.0.0-preview2-25226-02</CoreCLRVersion>
-       <JitVersion>2.0.0-preview2-25226-02</JitVersion>
+    <CoreFxCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreFxCurrentRef>
+    <CoreClrCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreClrCurrentRef>
+    <StandardCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</StandardCurrentRef>
   </PropertyGroup>
-
-  <!-- Create a collection of all project.json files for dependency updates. -->
+  
+  <!-- Auto-upgraded properties for each build info dependency. -->
+  <PropertyGroup>
+	<CoreFxExpectedPrerelease>preview2-25301-02</CoreFxExpectedPrerelease>
```

Comment:
(nit) can you clean up the whitespace in this file?  There is a mixture of tabs and spaces being used.  Use 2 spaces per indent (the default VS setting).